flo-scala_2.12/src/test/scala/com/spotify/flo/DslTest.scala

```diff
@@ -36,5 +39,27 @@ class DslTest extends FlatSpec with Matchers {
     exception should have message "Builder accessor used outside a defTask scope"
   }
 
+  it should "runInProcesses" in {
+    val foo = defTaskNamed("foo")
+      .process(() => {
+        val jvm = ManagementFactory.getRuntimeMXBean.getName
```

Comment:
Surprisingly, two processes run but this code seems to be executed in the `bar` process. The `foo` process seems to return a closure, which is then evaluated in the `bar` process.

😕

Reply:
Maybe I'm misunderstanding the syntax here...

Reply:
nvm my bad, I had obviously declared my process fn to return a closure :shrug: